osu.Game/Overlays/KeyBinding/GlobalKeyBindingsSection.cs

```diff
@@ -9,7 +10,12 @@ namespace osu.Game.Overlays.KeyBinding
 {
     public class GlobalKeyBindingsSection : SettingsSection
     {
-        public override IconUsage Icon => FontAwesome.Solid.Globe;
+        public override Drawable CreateIcon() => new SpriteIcon
+        {
+            RelativeSizeAxes = Axes.Both,
```

Comment:
I kind of dislike how the `RelativeSizeAxes` specification is repeated in every usage migrated from `SpriteIcon`. Removing it entirely (due to usage of `ConstrainedIconContainer`) or directly setting relative axes on the actual call in `SidebarButton` both work fine, so maybe these should just be removed in favour of either?

Reply:
You seem to have left a couple of in, but I think I got them all now.